docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
+$IsPhantom must be `'Y'` or `'y'` if it's a phantom file, else `'N'` or `'n'` 
```

Comment:
Can the single quotes be removed? 
(the pipeline input file doesn't require single quotes, not sure why this would)
i.e. Leave the ticks, changing:  `'Y'` to `Y` for all instances

Reply:
Yes, I will remove the single quotes.  

Based on one of Dave's comments I will switch Y to 1 and N to 0 as well since $IsPhantom is now going to expect a boolean.